### ISSUE TITLE
Encoding strategy change causes sadness with previously uploaded files?

### ISSUE THREAD

**Issue by FeminismIsAwesome** [carrierwaveuploader/carrierwave#2428]
Hello there,
so we have some file copies/uploads that point to stored AWS files and re-upload them to a new activerecord model. I noticed that a file that was previously named Company%2BLogo%2BHuge.png was getting converted to Company+Logo+Huge.png, which then meant it was unable to find the right file on aws with `image.remote_image_url = 'Company%2BLogo%2BHuge.png'` since it needed it with %2B, not encoded. This worked before on 1.3.1. How do you advise resolving this, given we have older aws files stored with % encoding like that?

tl;dr
`image.remote_image_url = 'aws.com/Company%2BLogo%2BHuge.png'` used to work, but now it will fail to find the right file since it encodes into + signs. Do we need to make our own downloader that maintains old behavior?

**Comment by mshibuya** [comment]
3faf7491e33bd10ae8b3e0010501fc96a76c21c3 should have fixed this.
